docs/release-notes/Percona-Everest-1.1.0-(2024-08-12).md

```diff
@@ -134,10 +134,16 @@ curl -sS "https://raw.githubusercontent.com/percona/everest-doc/main/tools/bin/c
 ```
 
 
-
 **Scenario 2**
 
-If a storage is currently in use by any schedule or backup, you can’t delete it. First, create a new backup or schedule with a different storage. Then, remove the old schedules and backups using the duplicate storage. Once it's no longer in use, you can delete it.
+If a storage is currently in use by any schedule or backup, you can’t delete it.
+
+Let's see how to work with different databases:
+
+- MySQL, Mongo: Create a new backup using a different storage. Then, delete the old schedules and backups which use the duplicated storages.
+
+- PostgreSQL: Delete the old schedules and backups that use the duplicate storage. Then, create a new backup using different storage.
```

Comment:
So if there are backups existing using two backup storages which are duplicate, then the backups using both the backup storages need to be deleted. Then delete the backup storage. Create the new backup storage, then take backups with the new backup storage.

Reply:
That's correct in terms of ideas, but with one addition
> then the backups using both the backup storages need to be deleted. 

 => then the backups _and schedules_ which use both backup storages need to be deleted. 

@mchawla16 do you suggest to change the mention `the duplicate storage` to `both duplicated storages`? 
I agree we should specify that, bc to be on the safe side it's not enough to cleanup one duplicate, both involved storages should be deleted. @rasika-chivate wdyt?

Reply:
This is more about style than substance.

You could consider adding a title to this scenario so it's easier to identify without having to read the entire description  ==>

> **Scenario 2**
> Details the steps to take when storage is in use by schedules or backups in various types of databases.

For the sentence of PostgreSQL ==>

> - PostgreSQL: The process is reversed: first, delete the old schedules and backups that use the duplicate storage. Then, create a new backup using different storage.

But I appreciate the team's feedback.

Reply:
Thanks @mchawla16 @edithturn @oksana-grishchenko for the feedback.